apps/client/src/app/pages/simulator/components/rotation-folder-page/rotation-folder-page.component.ts

```diff
@@ -54,19 +58,24 @@ export class RotationFolderPageComponent extends SeoPageComponent {
       }),
       shareReplay({ bufferSize: 1, refCount: true })
     );
-  }
 
-  protected getSeoMeta(): Observable<Partial<SeoMetaConfig>> {
-    return combineLatest([this.folder$, this.userId$]).pipe(
+    combineLatest([this.folder$, this.userId$]).pipe(
       map(([folder, userId]) => {
         return {
           title: `${folder.name} - FFXIV Teamcraft`,
           description: 'A list of crafting rotations.',
           url: `https://ffxivteamcraft.com/simulator/${userId}/rotation-folder/${folder.$key}`,
           image: 'https://ffxivteamcraft.com/assets/logo.png'
         };
-      }
-    ))
+      }),
+      takeUntilDestroyed(this.destroyRef)
+    ).subscribe(meta => {
+      this.seoService.setConfig(meta)
+    })
+  }
+
+  protected getSeoMeta(): Observable<Partial<SeoMetaConfig>> {
+    return
   }
```

Comment:
I don't think this is needed ^^

Reply:
Oops

Reply:
Oh that file appears to be outdated, I fixed that already.